control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml

```diff
@@ -69,15 +69,24 @@ spec:
             - name: SOURCE_DATA_PLANE_CONFIG_MAP_NAMESPACE
               value: knative-eventing
 
-            - name: BROKER_DATA_PLANE_CONFIG_MAP_NAME
+            - name: BROKER_CONTRACT_CONFIG_MAP_NAME
```

Comment:
Can we do this change in a separate PR? I don't think it is a good idea have this refactoring/renaming in this PR.

Reply:
Done in https://github.com/knative-sandbox/eventing-kafka-broker/pull/2602